components/Header.js

```diff
@@ -7,26 +7,32 @@ import {
   TextSize,
   Color,
 } from "@ableco/baseline";
+import titleize from "../utils/titleize";
 
-export function Header() {
+export function Header({ candidate }) {
   return (
     <Center className="mt-16">
       <Stack className="text-center" space={6}>
         <Center>
           <Avatar
-            source="https://assets.teenvogue.com/photos/5eac665515d49786070b2c49/1:1/w_1267,h_1267,c_limit/joe%20biden%20addresses%20allegations%20denies%20teen%20vogue.jpg"
+            source={candidate.foto}
             name="Joe Biden"
             width="104"
             height="104"
+            className="object-cover w-28 h-28"
```

Comment:
Since we use the `w-28 h-28` classes we could remove the width and height properties 👀